src/composer/src/build.rs

```diff
@@ -413,7 +482,7 @@ impl BuildState for DefaultBuilder {
         let binary = self.comp_obj_path(&c, &s)?;
         let argsfile = constructor_serialize_args(&c, &s, self)?;
         let tarfile = constructor_tarball_create(&c, &s, self)?;
-        let cmd = comp_gen_make_cmd(&binary, &argsfile, &tarfile, &c, &s);
+        let cmd = comp_gen_make_cmd(&binary, &argsfile, &tarfile, &c, &s, None);
```

Comment:
I would think that we'd want recompile for configured constants for the booter as well.

1. it enables us to be uniform, and 
2. it will enable us to customize things like # of CPUs entirely (eventually) in the configuration script.